main.js

```diff
@@ -0,0 +1,114 @@
+
+console.log('see me?');
+
+
+
+// declaring the variable first as an object, reference on line 24
+var taskInput = {};
+// taskToggleBtn is the object
+var taskToggleBtn = document.getElementById('taskToggle');
+var taskRunning = false;
+
+// timer stuff here - declare what it needs to do before func kicks off
+// first timer for 25 min
+
+var timerId = null;
+// how to render the # to the DOM: queries the DOM *******
+var h1 = document.querySelector('h1');
+
+//============
+// start first timer here
+var startTimer = 0;
+var number = 0;
+var timerId = null;
+// how to render the # to the DOM: queries the DOM *******
+var h1 = document.querySelector('h1');
+
+
+// THIS WORKS
+var number = 2501;
```

Comment:
This would be 2 and a half seconds.
`1000 milliseconds === 1 second` 
so 25 minuets would be 25 * 60 * 1000